ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/WelcomeFrame.kt

```diff
@@ -25,31 +25,53 @@ fun RemoteRobot.welcomeFrame(function: WelcomeFrame.() -> Unit) {
 @DefaultXpath("type", "//div[@class='FlatWelcomeFrame' and @visible='true']")
 class WelcomeFrame(remoteRobot: RemoteRobot, remoteComponent: RemoteComponent) : CommonContainerFixture(remoteRobot, remoteComponent) {
     fun openNewProjectWizard() {
-        actionLink(ActionLinkFixture.byTextContains("New Project")).click()
+        if (remoteRobot.ideMajorVersion() <= 202) {
+            actionLink(ActionLinkFixture.byTextContains("New Project")).click()
+        } else {
+            selectTab("Projects")
+            // This can match two things: If no previous projects, its a SVG icon, else a jbutton
+            findAll<ComponentFixture>(byXpath("//div[@accessiblename='New Project']")).first().click()
+        }
     }
 
     fun openPreferences() = step("Opening preferences dialog") {
-        actionLink("Configure").click()
-
-        // MyList finds both the list of actions and the most recently used file list, so get all candidates
-        val found = findAll(ComponentFixture::class.java, byXpath("//div[@class='MyList']"))
-            .any {
-                try {
-                    it.findText(remoteRobot.preferencesTitle()).click()
-                    true
-                } catch (e: NoSuchElementException) {
-                    false
+        if (remoteRobot.ideMajorVersion() <= 202) {
+            actionLink("Configure").click()
+
+            // MyList finds both the list of actions and the most recently used file list, so get all candidates
+            val found = findAll(ComponentFixture::class.java, byXpath("//div[@class='MyList']"))
+                .any {
+                    try {
+                        it.findText(remoteRobot.preferencesTitle()).click()
+                        true
+                    } catch (e: NoSuchElementException) {
+                        false
+                    }
                 }
-            }
 
-        if (!found) {
-            throw IllegalStateException("Unable to find ${remoteRobot.preferencesTitle()} in the configure menu")
+            if (!found) {
+                throw IllegalStateException("Unable to find ${remoteRobot.preferencesTitle()} in the configure menu")
+            }
+        } else {
+            selectTab("Customize")
+            findAndClick("//div[@accessiblename='All settings…']")
         }
+
         log.info("Successfully opened the preferences dialog")
     }
 
+    fun selectTab(tabName: String) {
+        if (remoteRobot.ideMajorVersion() <= 202) return
```

Comment:
nit: for all these checks `FIX_WHEN_MIN_IS_203 remove the if`?

Reply:
Sure, if this keeps getting bad, may make IDE version impls of an interface ...